geode-core/src/main/java/org/apache/geode/cache/client/ClientCacheFactory.java

```diff
@@ -536,6 +536,21 @@ public ClientCacheFactory setPoolSubscriptionRedundancy(int redundancy) {
     return this;
   }
 
+  /**
+   * A server has an inactivity monitor that ensures a message is sent to a client at least once a
+   * minute (60,000 milliseconds). If a subscription timeout multipler is set in the client it
+   * enables timing out of the subscription feed with failover to another server.
+   * <p>
+   * A value of zero (the default) disables timeouts
+   * <p>
+   * The resulting timeout will be multiplied by 1.25 in order to avoid race conditions with the
```

Comment:
Can we clarify what a user should actually set this field to? Maybe say something like 

"The client will time out it's subscription connection after a number of seconds equal to this multiplier times the server's subscription-timeout. 

Set this to 2 or more to make sure the client will receive pings from the server before the timeout."

Reply:
@upthewaterspout - I'll add this to the javadocs for this method and the PoolFactory method but I won't push the changes since it will kick off another precheckin run.